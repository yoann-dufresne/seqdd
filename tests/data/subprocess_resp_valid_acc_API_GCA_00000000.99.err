  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   259  100   259    0     0   2468      0 --:--:-- --:--:-- --:--:--  2466
