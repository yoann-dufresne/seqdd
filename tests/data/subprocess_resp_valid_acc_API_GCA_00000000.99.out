<?xml version="1.0" encoding="UTF-8"?>
<ErrorDetails>
  <timestamp>1732893043100</timestamp>
  <status>404</status>
  <error>Not Found</error>
  <message>GCA_00000000.99 not found.</message>
  <path>/ena/browser/api/xml/GCA_00000000.99</path>
</ErrorDetails>
